services/121-service/src/payments/fsp-integration/intersolve-visa/intersolve-visa.service.ts

```diff
@@ -796,4 +704,24 @@ export class IntersolveVisaService
       return 0;
     }
   }
+
+  public async pauseCardOrThrow(tokenCode: string, blockToken: boolean) {
```

Comment:
Is it our practice that any method where we deliberately throw an error if something goes wrong suffix it with "OrThrow"? If so, then I have some refactoring to do :)